### ISSUE TITLE
email addresses in text content elements are not obfuscated anymore

### ISSUE THREAD

**Issue by fritzmg** [contao/core-bundle#594]
### Reproduction

1. create a new text content element
2. insert `<a href="mailto:example@example.org">example@example.org</a>` (via source code)
3. save, open the page in the frontend and view the source code

#### Expected behaviour

In Contao 3, the following HTML code will be produced:
```html
<a href="&#109;&#97;&#105;&#108;&#116;&#111;&#58;&#x65;&#x78;&#97;&#x6D;&#112;&#108;&#x65;&#x40;&#x65;&#120;&#97;&#109;&#x70;&#x6C;&#x65;&#46;&#x6F;&#x72;&#103;">&#x65;&#x78;&#97;&#x6D;&#112;&#108;&#x65;&#x40;&#x65;&#120;&#97;&#109;&#x70;&#x6C;&#x65;&#46;&#x6F;&#x72;&#103;</a>  
```

#### Actual behaviour

In Contao 4.2.4, the following HTML code will be produced:
```html
<a href="&#109;&#97;&#105;&#108;&#116;&#111;&#58;example@example.org">example@example.org</a>
```
Only the `mailto:` part of the `href` attribute is obfuscated for some reason.

**Comment by xchs** [comment]
I was able to reproduce the issue, too.

**Comment by leofeyer** [comment]
Fixed in 59fe372533391abbee94aaa8ff07350cd51876f7.
